foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/generator/DataRecordsModuleStep.java

```diff
@@ -237,6 +267,72 @@ public void the_spark_functionality_is_generated_in_the_module() {
         assertTrue("Schema class not created: " + schemaClass, Files.exists(schemaClass));
     }
 
+    @Then("the {string}, {string}, and {string} configurations are generated to access the {string} file store")
+    public void the_and_configurations_are_generated_to_access_the_file_store(String fsProvider, String fsAccessKeyId, String fsSecretAccessKey, String fileStoreName) throws IOException {
+        AtomicBoolean hasFsProviderEnvVar = new AtomicBoolean(false);
+        AtomicBoolean hasFsAccessKeyIdEnvVar = new AtomicBoolean(false);
+        AtomicBoolean hasFsSecretAccessKeyEnvVar = new AtomicBoolean(false);
+
+        String pipelineSparkBaseValuesYamlFileName = PipelineUtils.deriveArtifactIdFromCamelCase(pipeline.getName()) + "-base-values.yaml";
+        File pipelineSparkBaseValuesYamlFile = new File(projectDir.toString() + "/main/resources/apps/" + pipelineSparkBaseValuesYamlFileName);
+
+        assertTrue("The " + pipelineSparkBaseValuesYamlFileName + " file was not generated!", pipelineSparkBaseValuesYamlFile.exists());
+
+        Files.lines(Paths.get(pipelineSparkBaseValuesYamlFile.getAbsolutePath())).forEach(line -> {
+            if(line.contains("- name: \"" + fsProvider + "\"")) {
+                hasFsProviderEnvVar.set(true);
+            }
+
+            if(line.contains("- name: \"" + fsAccessKeyId + "\"")) {
+                hasFsAccessKeyIdEnvVar.set(true);
+            }
+
+            if(line.contains("- name: \"" + fsSecretAccessKey + "\"")) {
+                hasFsSecretAccessKeyEnvVar.set(true);
+            }
+        });
+
+        assertTrue("File Store Env variable \"" + fsProvider + "\" not found in " + pipelineSparkBaseValuesYamlFile, hasFsProviderEnvVar.get());
```

Comment:
S: Might be able to change this to `assertEquals` and de-duplicate the code between this and the negative assertion below

Reply:
Refactored.